packages/atomic/src/components/facets/facet-common.ts

```diff
@@ -88,6 +89,8 @@ type DisplayValuesAsProp = {
 
 type CollapsedProp = {isCollapsed?: boolean};
 
+type HeadingLevelProp = {headingLevel?: number};
```

Comment:
Should it be mandatory on all types of facets ? Even segmented facets ?

If so, it should probably be non optional, and part of the `props on all facets` type.

The goal of these types is to make it obvious when you create a new type of facet what are all the options you should be implementing, so that there's standardization across very similar components.

We should try to make as much option non optional, when it's possible to do so.

Reply:
I don't think it should be mandatory for segmented facets